dev/PushNotifications/PushManager.h

```diff
@@ -11,6 +14,12 @@ namespace winrt::Microsoft::ProjectReunion::implementation
         PushManager() = default;
 
         static Windows::Foundation::IAsyncOperationWithProgress<Microsoft::ProjectReunion::ChannelResult, Microsoft::ProjectReunion::ChannelResult> CreateChannelAsync(winrt::guid remoteId);
+
+    private:
+        static bool isChannelRequestRetryable(const winrt::hresult& hrException);
+        static std::list<winrt::guid> remoteIdList;
```

Comment:
Why `list` rather than others, say, `vector`?

Reply:
+1 Vector is sufficient here.

Reply:
Again this PR was meant to be a prototype and will be optimized. So will use a vector going ahead. Thanks.